kyo-prelude/shared/src/main/scala/kyo/kernel/ArrowEffect.scala

```diff
@@ -192,19 +408,24 @@ object ArrowEffect:
         handleLoop(state, v, Context.empty)
     end handleState
 
-    inline def handleCatching[I[_], O[_], E <: ArrowEffect[I, O], A, B, S, S2, S3](
-        inline tag: Tag[E],
+    private[kyo] inline def handleCatching[I[_], O[_], E <: ArrowEffect[I, O], A, B, S, S2, S3](
```

Comment:
This functionality can be achieved indirectly via `Effect.catching` + `ArrowEffect.handle` and the API isn't very safe given that it has `accept` so I've made the method `private[kyo]`.